fuzzers/005-tilegrid/generate_full.py

```diff
@@ -451,9 +451,9 @@ def run(json_in_fn, json_out_fn, verbose=False):
     propagate_INT_lr_bits(database, tiles_by_grid, verbose=verbose)
     propagate_INT_bits_in_column(database, tiles_by_grid)
     propagate_rebuf(database, tiles_by_grid)
-    propagate_IOB_SING(database, tiles_by_grid)
-    propagate_IOI_SING(database, tiles_by_grid)
-    propagate_IOI_Y9(database, tiles_by_grid)
+    #propagate_IOB_SING(database, tiles_by_grid)
```

Comment:
Put this behind a `os.getenv('XRAY_PART') != 'spartan7'`

Reply:
Yah, I can do that -- however, that's just a work-around. There are in fact the "SING" blocks on the top and bottom of each column, but it looks like the mapping is just slightly different from what is done on the Artix. I couldn't understand the code inside propagate_IOB_SING well enough though to figure it out, so I just commented it out to get the fuzzers to start running.

Given that, is there a more clever way to flag this as a "todo" item, rather than just commenting it out (which might lead us to forget that this needs to be fixed down the road)?